### ISSUE TITLE
[Story] Store and return template display name

### ISSUE THREAD

**Issue by marrobi** [microsoft/AzureTRE#708]
## Description

As a UI developer
I want the API to return the friendly name of a workspace template 
So that I can display it to users

## Acceptance criteria

- [ ] Extract title from template schema json document and sent in registration payload as `display_name`
- [ ] `display_name` stored in state store 
- [ ] API operations for get template/s returns returns the title as `display_name` in addition tot he existing `name` field.

**Comment by tamirkamara** [comment]
@marrobi should we consider using the same terminology for _title_ and _display_name_?

**Comment by marrobi** [comment]
@tamirkamara  Yes, did think about it and probably a good idea. The JSON schema has to be title however we use display_name in various places already in the API.

We could settle on title throughout?
